docs/src/opensourcetpa.md

```diff
@@ -0,0 +1,294 @@
+# Open source TPA
+
+Trusted Postgres Architect has historically been built to address the deployment needs of EDB customers. Among its playbooks is the ability to deploy EDB-recommended architectures using EDB repositories containing EDB versions of Postgres and other tools.
+
+As an open source deployment tool, TPA can perform similar tasks using open source repositories. This section of the documentation covers what an open source user of TPA needs to know: How to set up and configure TPA and deploy a basic configuration of open source Postgres tools.
+
+## Setting up TPA
+
+### Prerequisites
+
+Before you install TPA, you must install the required packages: Python 3, git, openvpn and patch.
+
+#### Debian
+
+```
+sudo apt-get install python3.7 python3-pip python3-venv git openvpn patch
+```
+#### Ubuntu
+
+```
+sudo apt-get install python3.6 python3-pip python3-venv git openvpn patch
+
+```
+
+#### Redhat, Rocky or AlmaLinux (RHEL7)
+
+```
+sudo yum install python3 python3-pip epel-release git openvpn patch
+```
+
+#### Redhat, Rocky or AlmaLinux (RHEL8)
+
+```
+sudo yum install python36 python3-pip epel-release git openvpn patch
+```
+
+#### macOS
```

Comment:
We don't support macOS so it's questionable whether this should be here at all (same for the 'install from source' page). At minimum this should have a warning.

Reply:
Removed from here.